### PR TITLE
align nnet decoder & refactor

### PR DESCRIPTION
**Pull request by SmileGoat**
### PR types

[New features]
### Describe

align nnet decoder & refactor

### CODE REVIEW

File: speechx/examples/decoder/offline_decoder_main.cc [hunk 1]
```diff
@@ -17,50 +17,75 @@
...
+#include "frontend/raw_audio.h"
 #include "kaldi/util/table-types.h"
 #include "nnet/decodable.h"
 #include "nnet/paddle_nnet.h"
 
-DEFINE_string(feature_respecifier, "", "test nnet prob");
+DEFINE_string(feature_respecifier, "", "test feature rspecifier");
+DEFINE_string(model_path, "avg_1.jit.pdmodel", "xxx.pdmodel");
+DEFINE_string(param_path, "avg_1.jit.pdiparams", "xxx.pdiparams");
+DEFINE_string(dict_file, "vocab.txt", "xxx.pdmodel");
```
Comment (zh794390558) [inline speechx/examples/decoder/offline_decoder_main.cc]: help info

File: speechx/examples/decoder/offline_decoder_main.cc [hunk 2]
```diff
@@ -17,50 +17,75 @@
...
+DEFINE_string(lm_path, "lm.klm", "xxx.pdiparams");
```
Comment (zh794390558) [inline speechx/examples/decoder/offline_decoder_main.cc]: help info

File: speechx/speechx/frontend/raw_audio.h [hunk 3]
```diff
@@ -18,13 +18,15 @@
...
+#pragma once
+
 namespace ppspeech {
 
 class RawAudioCache : public FeatureExtractorInterface {
   public:
     explicit RawAudioCache(int buffer_size = kint16max);
     virtual void Accept(const kaldi::VectorBase<BaseFloat>& waves);
-    virtual bool Read(kaldi::Vector<kaldi::BaseFloat>* waves);
+    virtual bool Read(kaldi::Vector<kaldi::BaseFloat>* feat);
```
Comment (zh794390558) [inline speechx/speechx/frontend/raw_audio.h]: waves?

Comment (SmileGoat) [reply speechx/speechx/frontend/raw_audio.h]: waves，改一下。

Comment (SmileGoat) [reply speechx/speechx/frontend/raw_audio.h]: 估计文件改错了

**Pull request merged**